src/py/aspen/app.py

```diff
@@ -55,7 +55,7 @@ def decorated(*args, **kwargs):
 # else send index.html. Allows reloading React app from any route.
 @application.route("/", defaults={"path": ""})
 @application.route("/<path:path>")
-@requires_auth
+#@requires_auth
```

Comment:
is this change intentional?

Reply:
no i meant to remove that, since it doesn't fit in with the v0 user flow